sgid_client/util.py

```diff
@@ -45,4 +47,41 @@ def safe_json_parse(json_string: str) -> dict | list | str:
     try:
         return json.loads(json_string)
     except Exception:
-        return json_string
\ No newline at end of file
+        return json_string
+
+
+def isSgidUserinfoObject(data: object) -> bool:
+    """Checks whether the input is a valid sgID userinfo object by checking:
+    1. Whether the input is a defined dictionary
+    2. That all keys and values in the dictionary are strings
+
+    Args:
+        dataValue (object): an unknown object
+
+    Returns:
+        A boolean that indicates whether the input was a valid sgID userinfo object
+    """
+    if type(data) is not dict:
+        return False
+
+    result = True
+    for key, value in data.items():
+        if type(key) is not str or type(value) is not str:
+            result = False
+
+    return result
+
+
+def parseIndividualDataValue(dataValue: str) -> dict | list | str:
```

Comment:
same comment as above. `parse_individual_data_value`?

Reply:
fixed in [396ad25](https://github.com/opengovsg/sgid-client-python/pull/42/commits/396ad257e153840628f4ebbbc22b7676f1c37cfd)